transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/DefaultMigrationManager.java

```diff
@@ -39,27 +43,37 @@
           new Migration(
               4,
               "Add flush index",
-              "CREATE INDEX IX_TXNO_OUTBOX_1 ON TXNO_OUTBOX (processed, blacklisted, nextAttemptTime)"));
+              "CREATE INDEX IX_TXNO_OUTBOX_1 ON TXNO_OUTBOX (processed, blacklisted, nextAttemptTime)"),
```

Comment:
I would have thought `ALTER` is the default, no?

Reply:
not too sure what you mean? the default specified here is for the mysql dialects, since it looked like postgres9 and h2 had slightly different syntax for alter anyway.

Reply:
Ah yes, does look like modify is more common. Learned a thing :)